ci_framework/roles/libvirt_manager/molecule/deploy_edpm_compute/converge.yml

```diff
@@ -0,0 +1,24 @@
+---
+# Copyright 2023 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Converge
+  hosts: all
+  gather_facts: true
+  vars:
+    ansible_user_dir: "/tmp"
+    cifmw_libvirt_manager_installyamls: "/tmp/install_yamls"
```

Comment:
so you'll be able to point to:
`"{{ ansible_user_dir }}/{{ zuul.projects['github.com/openstack-k8s-operators/install_yamls'].src_dir }}"`

Reply:
Good to know. I'll keep in mind for future references to the `install_yamls`